lib/commands/moveToObject.js

```diff
@@ -59,6 +28,10 @@ let moveToObject = function (selector, xoffset, yoffset) {
             throw new RuntimeError(7)
         }
 
+        if (this.isMobile) {
```

Comment:
This is probably cleaner to do inside the moveTo() method

Reply:
I went ahead and move is call into the moveTo() method. I'm not sure it made it cleaning though with how the moveTo method checks for valid arguments compared to how the touchMove() check for valid args. Let me know what you think.